### PR TITLE
build: update alpine in docker build to version 3.14

### PR DESCRIPTION
**Pull request by eworm-de**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Run 'make docker' to build a statically linked vis executable!
-FROM i386/alpine:3.13
```
Comment (mcepl) [inline Dockerfile:2]: Wouldn’t it be better to have here `alpine:latest`?

**Review by mcepl — CHANGES_REQUESTED** [review]
Looks completely trivial.

File: Dockerfile (hunk 1 above)
Comment (eworm-de) [reply Dockerfile:2]: Not sure... I think the idea here was to have reproducibility. However packages are updated within one branch as well.

Also we had breaking changes with alpine updates, see 0ba252f08e9ef7b62c442eb91b36822d654f5b04 for example. With a commit for every version an issue shows up in bisecting.

Comment (mcepl) [reply Dockerfile:2]: :+1: then, but it should be the Biblical 3.16 now, shouldn't it?

**Review by mcepl — APPROVED** [review]
It wouldn't hurt, right?

File: Dockerfile (hunk 1 above)
Comment (eworm-de) [reply Dockerfile:2]: Last commit is for 3.16.
Let's keep everything between, again for bisecting.

**Comment by deepcube** [comment]
On Wed Jun 1, 2022 at 9:32 AM MDT, Christian Hesse wrote:
> Well, you squashed it... So no bisecting between apline versions possible...
> Thanks anyway!

Sorry about that, I missed the comments about bisecting.  Next time!

**Comment by eworm-de** [comment]
I am not aware of any breakage, so let's hope this is fine. 😜

**Pull request closed without merging**